server/src/controllers/Venue/resolver.ts

```diff
@@ -102,6 +104,85 @@ export class VenueResolver {
     @Arg('_onlyUsedForAuth', () => Int) _onlyUsedForAuth: number,
   ): Promise<{ id: number }> {
     // TODO: handle deletion of non-existent venue
+    const users = await prisma.users.findMany({
+      where: {
+        user_events: {
+          some: {
+            event: {
+              canceled: false,
+              ends_at: { gt: new Date() },
+              venue_id: id,
+            },
+            subscribed: true,
+            attendance: { name: 'yes' },
+          },
+        },
+      },
+      include: {
+        user_events: {
+          include: { event: true },
+          where: {
+            event: {
+              canceled: false,
+              ends_at: { gt: new Date() },
+              venue_id: id,
+            },
+            subscribed: true,
+            attendance: { name: 'yes' },
+          },
+        },
+      },
+    });
+
+    const venue = await prisma.venues.findUniqueOrThrow({
+      where: { id: id },
+    });
+
+    const eventList = (event: events[], currentUserId: number) => {
+      event.map(
+        ({ name, id }) => `
+      <tr>
+      <td>${name}</td>
+      <td>${eventUnsubscribeOptions({
+        eventId: id,
+        userId: currentUserId,
+      })}</td>
+     </tr>
+  `,
+      );
+    };
+    const emailSubject = `Events hosted at ${venue.name} won't be hosted there anymore`;
+    const emailContent = (
+      event: events[],
+      currentUserId: number,
+    ) => `The events related to ${
+      venue.name
+    } won't be host locally anymore.<br />
+    <table>
+    <thead>
+        <tr>
+          <th>Event</th>
+          <th>Cancel RSVP</th>
+        </tr>
+    </thead>
+    <tbody>
+       ${eventList(event, currentUserId)}
```

Comment:
```suggestion
      ${eventList(event, currentUserId).join('')}
```
By default array will be in output with commas as separators.

Reply:
Nice catches, thank you 💚